prime-router/src/main/kotlin/cli/ValidateTranslationSchemaCommand.kt

```diff
@@ -0,0 +1,73 @@
+package gov.cdc.prime.router.cli
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.types.enum
+import gov.cdc.prime.router.azure.BlobAccess
+import gov.cdc.prime.router.fhirengine.translation.TranslationSchemaManager
+import org.apache.logging.log4j.kotlin.Logging
+
+class ValidateTranslationSchemaCommand :
+    CliktCommand(
+    name = "validateSchemas",
+    help = "Commands to validate translation schemas"
+),
+    Logging {
+
+    private val translationSchemaManager = TranslationSchemaManager()
+
+    /**
+     * The schema type to validate
+     */
+    private val schemaType: TranslationSchemaManager.SchemaType by option(
+        "--schema-type",
+        "-s",
+        help = "which schema type to process"
+    ).enum<TranslationSchemaManager.SchemaType>().required()
+
+    private val blobStoreConnection: String by option(
```

Comment:
Could the blob store info be deduced from the environment? Most CLI commands pass in the environment (staging, prod, local) and perform based on that.

Reply:
Potentially, though I think it might be limiting to do it that way since this command might need to get run across a couple environments